arch/cortex-m/src/nvic.rs

```diff
@@ -207,9 +209,9 @@ pub unsafe fn has_pending() -> bool {
 /// Returns whether there are any pending interrupt bits set while ignoring
 /// the indices that correspond to the bits set in mask
 ///
-/// Mask is defined as two u128 fields,
-///   mask.0 has the bits corresponding to interrupts from 128 to 240
-///   mask.1 has the bits corresponding to interrupts from 0 to 127
+/// Mask is defined as two `u128` fields,
+///   `mask.0` has the bits corresponding to interrupts from 128 to 240
+///   `mask.1` has the bits corresponding to interrupts from 0 to 127.
```

Comment:
```suggestion
///   `mask.0` has the bits corresponding to interrupts from 128 to 240.
///   `mask.1` has the bits corresponding to interrupts from 0 to 127.
```
I sorta get the 'trailing sentence' thing going from the comma above, but I think this makes more sense? I don't feel strongly at all; commit the suggestion or resolve it without.